src/utility/util.js

```diff
@@ -0,0 +1,33 @@
+const { text } = require("express");
+const fs = require("fs");
+const path = require("path");
+
+const getNotesList = () => {
+  // get manga data fom the db file by reading, then parse the string and re.json
+
+  const notesJSON = fs.readFileSync(
+    path.join(__dirname, "../../db/db.json"),
+    "utf-8"
+  );
+
+  const notes = JSON.parse(notesJSON);
+
+  return notes;
+};
+
+const identifyNoteId = (notesDbArray, id) => {
+  return notesDbArray.find((each) => {
+    return each.id == id;
+  });
+};
+
+const validKeys = ["title", "text"];
```

Comment:
Doesn't need to be declared separately. We only need to do this with functions that will be used in multiple places, or perhaps very large variables which are taking up a lot of space to clean up our code visually. This should be declared in the function in which it is required.